src/seedu/addressbook/Main.java

```diff
@@ -79,14 +79,20 @@ private void exit() {
     /** Reads the user command and executes it, until the user issues the exit command.  */
     private void runCommandLoopUntilExitCommand() {
         Command command;
-        do {
-            String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
-
-        } while (!ExitCommand.isExit(command));
+        try{
+        	this.storage.checkFileNotMissing();
```

Comment:
This may be a too high-level method to check for this low-level problem. `StorageFile::save()` may be a more suitable place to detect it. However, this is a suitable place to handle the exception thrown from `StorageFile::save()`